Tasks/FtpUploadV2/task.json

```diff
@@ -148,6 +157,19 @@
             "required": true,
             "helpMarkDown": "Selecting this option results in the FTP server's SSL certificate being trusted with ftps://, even if it is self-signed or cannot be validated by a Certificate Authority (CA).",
             "groupName": "advanced"
+        },
+        {
+            "name": "customCmds",
+            "type": "multiLine",
+            "label": "Optional FTP Commands",
```

Comment:
I would remove 'Optional' from the label.

help: `Optional commands to customize the FTP connection.`